File: eval/fixtures/legacy-directory-backed-asmx-commerce/src/Legacy/Billing/BillingRefundManager.cs
namespace Acme.Legacy.Billing;

public class BillingRefundManager
{
  private readonly BillingRefundRepository _repository = new BillingRefundRepository();

  public string RebuildRefundAudit(string refundId)
  {
    return _repository.WriteRefundAuditEntry(refundId);
  }
}

File: eval/fixtures/legacy-directory-backed-jaxws-commerce/src/main/java/com/acme/legacy/billing/BillingRefundManager.java
package com.acme.legacy.billing;

public class BillingRefundManager {
  private final BillingRefundRepository billingRefundRepository = new BillingRefundRepository();

  public RefundAuditResult rebuildRefundAudit(String refundId) {
    return billingRefundRepository.writeRefundAuditEntry(refundId);
  }
}
